.github/workflows/release.yml

```diff
@@ -0,0 +1,59 @@
+name: release
```

Comment:
Nice! This does way more than expected 👍. Going to have a look at this tomorrow since I have a different setup for my repositories handling things in here. If you're interested you can see it at: https://github.com/WyriHaximus/php-test-utilities/tree/master/.github/workflows👍

Reply:
@WyriHaximus Yea I think @haya14busa has some really helpful actions. Thanks for a link to your actions there are some really useful ones in there (changelog-generator and milestones).

Reply:
@rickstaa Yeah, there are some useful things in there! Really need to play with it before merging it, and will probably change some things afterwards. Will keep you posted